vocab/security/vocabulary.yml

```diff
@@ -3,6 +3,8 @@ vocab:
   value: https://w3id.org/security#
 
 prefix:
+  - id: rdf
+    value: http://www.w3.org/1999/02/22-rdf-syntax-ns#
```

Comment:
Strictly speaking, this is not necessary (although it does not harm). The tool has a number default prefixes which include rdf, rdfs, dc, owl, xsd.

```suggestion
```

Reply:
Ah! Thank you, I checked the yml2vocab tool and didn't see the definition. Great that it exists, removing the `rdf` definition now.